configuration/concepts/lab_tests.csv

```diff
@@ -95,27 +100,76 @@ d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Co
 1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,
 dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,
 acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,PH; Specific gravity; Glucose; Ketones; Urobilinogen; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,
-4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-74ab27c4-c745-45ff-b7e6-e8dbaf933458,,,Trichomonas,Trichomonas,Trichomonas,Trichomonas,,LabTest,Coded,,Positive; Negative;,,,,,,,,
+4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Coded,,Positive; Negative,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Coded,,Positive; Negative,,,,,,,,
+74ab27c4-c745-45ff-b7e6-e8dbaf933458,,,Trichomonas,Trichomonas,Trichomonas,Trichomonas,,LabTest,Coded,,Positive; Negative,,,,,,,,
 14e99cf4-d294-4905-8837-40e704d8a6ad,,,Vaginal Candidiasis,Candidiasis vaginal,Candidiasis vaginal,Vaginal Candidiasis,,LabTest,Text,,,,,,,,,,
-ab6588f6-87ee-45ca-a6c5-dc1337f83469,,,Gonorrhea,Gonorrea,Gonorrea,Gonorrhea,,LabTest,Coded,,Positive; Negative;,,,,,,,,
-637b4f48-8717-45e2-a5f8-96869e3bb768,,,Chlamydia,Clamidia,Clamidia,Chlamydia,,LabTest,Coded,,Positive; Negative;,,,,,,,,
+088253e3-b556-4e82-b646-2bb784a46577,,,Blood HCG,HCG en sangre,HCG en sangre,Blood HCG,,LabTest,Coded,,Positive; Negative,,,,,,,,

```

Comment:
We said we change this to **Blood HCG (Pregnancy)**

Reply:
Done